src/test/javascript/spec/component/modeling-exercise/modeling-exercise.component.spec.ts

```diff
@@ -61,14 +61,14 @@ describe('ModelingExercise Management Component', () => {
         comp.modelingExercises = [modelingExercise];
     });
 
-    afterEach(function () {
+    afterEach(() => {
         jest.restoreAllMocks();
     });
 
     it('Should call loadExercises on init', () => {
         // GIVEN
         const headers = new HttpHeaders().append('link', 'link;link');
-        const findStub = jest.spyOn(courseExerciseService, 'findAllModelingExercisesForCourse').mockReturnValue(
+        const findSpy = jest.spyOn(courseExerciseService, 'findAllModelingExercisesForCourse').mockReturnValue(
```

Comment:
This is not a spy though. It returns the same default value for all calls. Therefore "Stub" is correct